pcp/versions/2.0/inventory.md

```diff
@@ -63,8 +99,8 @@ Error Handling
 
 In case of an error during the processing of the inventory list, the broker must
 reply with an error message as described in the [error handling][2] section.
-The *message_type* field of the JSON data `content` must be equal to the
-inventory request one mentioned above.
+The *message_type* field of the JSON data `content` must be equal to
+`http://puppetlabs.com/inventory_request`.
 
 Server Operation
```

Comment:
I'd note under server operation that if an inventory request with subscribe=false is received after one with subscribe=true, the broker should stop sending updates.